src/platform/utilities/environment/stagingDomains.js

```diff
@@ -0,0 +1,63 @@
+// This list also exists in script/options.js
+const domainReplacements = [
+  { from: 'www\\.cem\\.va\\.gov', to: 'staging.cem.va.gov' }
+];
```

Comment:
I think this data would have to be passed through the `create-build-settings.js` middleware and referenced through the window. We probably need some sort of module to make lookups to that data more centralized, versus global access.

Reply:
I'm going to explore some other ideas, but at the moment, that might be the method of passing data.